obal/__init__.py

```diff
@@ -152,4 +153,8 @@ def main():
 
 
 if __name__ == '__main__':
+    # this needs to be outside of main, as otherwise PlaybookCLI fails
+    # to set the verbosity correctly
+    display = Display()
```

Comment:
This variable is unused. Either you can remove the variable (not the actual function) or you should use it.

Reply:
It's a tad more complicated than this. `PlaybookCLI` expects an `display` object to exist at the global scope, and modifies its `verbosity` attribute. If there is none, it will create it, but then the settings are not propagated and you end up with non-verbose output.

Reply:
I meant that ```Display()``` should be sufficient if you don't use the ```display``` variable but if it needs to exist at the global scope that's a good comment to add. pylint would complain about this. An alternative is to use ```global display = Display()``` in ```main()``` since that's explicit.

Reply:
```
    global display = Display()
                   ^
SyntaxError: invalid syntax
```

Reply:
ah, it works in two lines.